test/e2e/node/runtimeclass.go

```diff
@@ -59,7 +59,9 @@ var _ = SIGDescribe("RuntimeClass", func() {
 		}
 		_, err = f.ClientSet.CoreV1().Pods(f.Namespace.Name).Create(context.TODO(), pod, metav1.CreateOptions{})
 		framework.ExpectError(err, "should be forbidden")
-		framework.ExpectEqual(apierrors.IsForbidden(err), true, "should be forbidden error")
+		if !apierrors.IsForbidden(err) {
+			framework.Failf("should be forbidden error")
```

Comment:
Include more information here. What is the actual value?

```suggestion
			framework.Failf("expected 'forbidden' as error, got instead: %v", err)
```